prometheus-collector/MAC-3P-Docs/Instructions for Private Preview Onboarding.md

```diff
@@ -1,79 +1,115 @@
-# Instructions for Private Preview onboarding
+# Instructions for Azure Managed Service for Prometheus Private Preview onboarding
 
-**1. The steps need to executed in this order. If not, step 1 has to be rerun after executing step 2**
+**1. The steps need to be executed in the order shown below.**
```

Comment:
Scott, we might probably want to keep the statement - "If not, step 1 has to be rerun after executing step 2", just to make it obvious?

Reply:
added info back in about re-running step 1